debian/dosemu2-dosinstaller.install

```diff
@@ -0,0 +1,7 @@
+usr/bin/dosemu-downloaddos
+usr/bin/dosemu-installfreedos
+usr/bin/dosemu-installfreedosuserspace
+usr/share/dosemu/libexec/dosemu-downloaddos
+usr/share/dosemu/libexec/dosemu-installfreedosuserspace
+usr/share/dosemu/dosemu2-cmds-0.2/dosemu/insfdusr.bat
+usr/share/dosemu/dosemu2-cmds-0.2/dosemu/dosrc.d/sugfdusr.bat
```

Comment:
does the `dosemu2-cmds-0.2` directory still exist?

Reply:
`commands` please.